ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/component/StateComponent.java

```diff
@@ -41,6 +41,6 @@
      *
      * @param state the component state object
      */
-    void loadState(@NotNull JsonObject state);
+    void setState(@NotNull JsonObject state);
```

Comment:
Actually it changes the state, it sets a new state.
But it's not a problem to me to revert the method name.